docs/CRSF_SENSORS.md

```diff
@@ -0,0 +1,40 @@
+# mLRS Documentation: CRSF Sensors #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+mLRS provides the following telemetry sensors when using CRSF:
+
+Num | Datapoint | Description                                                              | Data Source |
+--- | --------- | ------------------------------------------------------------------------ | ----------- |
+1   | 1RSS       | Receiver RSSI                                                           | mLRS        |
```

Comment:
I think we want to mention that it is the negative of the RSSI! (to avoid possible confusion with other sources of RSSI)
also applies to RSSI's below
maybe a footnote?

Reply:
Added note: 'RSSI values are displayed as positive values, e.g. -85dBm is displayed as 85dBm'